formula/src/main/java/com/instacart/formula/internal/ChildrenManager.kt

```diff
@@ -12,7 +11,7 @@ internal class ChildrenManager(
     private val delegate: FormulaManagerImpl<*, *, *>,
     private val inspector: Inspector?,
 ) {
-    private var children: SingleRequestMap<Any, FormulaManager<*, *>>? = null
+    private val children: SingleRequestMap<Any, FormulaManager<*, *>> = LinkedHashMap()
```

Comment:
No need to optimize `children` map initialization as `FormulaManager` optimizes this already